cmd/ocm/gcp/delete-wif-config.go

```diff
@@ -59,21 +58,23 @@ func deleteWorkloadIdentityConfigurationCmd(cmd *cobra.Command, argv []string) {
 		log.Fatal("WIF config ID is required")
 	}
 
-	// Create clients
-	ocmClient, err := alphaocm.NewOcmClient()
+	// Create the client for the OCM API:
+	connection, err := ocm.NewConnection().Build()
 	if err != nil {
-		log.Fatalf("failed to create backend client: %v", err)
+		log.Fatal(err)
```

Comment:
isnt this missing a return stmt when error occurs ?

Reply:
I thought previously discussed bubbling up the errors to the caller and allow the top level "Run" funtion decide how to handle/display the error.

I dont see these kind of Fatal log msgs anywhere in the else in the ocm or rosa cli code

Reply:
An example error:
```
➜  go run ./cmd/ocm gcp delete wif-config 2crdacmjbo3jmdgfn4dq791a7nig04qg
2024/07/30 14:25:07 Not logged in, credentials aren't set, run the 'login' command
exit status 1
```
This is the top level Run function.